docker/config/local_dev.env

```diff
@@ -38,7 +38,7 @@ resource.metrics.markus_backends=markus.backends.logging.LoggingMetrics,markus.b
 # sentry
 # ------
 
-# secrets.sentry.dsn=
+secrets.sentry.dsn=http://public@fakesentry:8090/1
```

Comment:
These are all from the point-of-view of the webapp, crontabber, and processor containers.